bindings/pyroot/test/pretty_printing.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import ROOT
+
+
+class PrettyPrinting(unittest.TestCase):
+    # Helpers
+    def _print(self, obj):
+        print("{}: {}".format(obj.__cppname__, obj))
+
+    # Tests
+    def test_RVec(self):
+        v = ROOT.VecOps.RVec("float")(4)
+        for i in range(v.size()):
+            v[i] = i
+        self._print(v)
+        self.assertIn("{ 0", v.__str__())
+
+    def test_STLVector(self):
+        v = ROOT.std.vector("float")(4)
+        for i in range(v.size()):
+            v[i] = i
+        self._print(v)
+        self.assertIn("{ 0", v.__str__())
+
+    def test_STLMap(self):
+        m = ROOT.std.map("string", "int")()
+        for i, s in enumerate(["foo", "bar"]):
+            m[s] = i
+        self._print(m)
+        self.assertIn("foo", m.__str__())
+        self.assertIn("bar", m.__str__())
+
+    def test_STLPair(self):
+        p = ROOT.std.pair("string", "int")("foo", 42)
+        self._print(p)
+        self.assertIn("foo", p.__str__())
+
```

Comment:
Hi @stwunsch , this looks great! I wonder if it would make sense to add some more test(s) for classes that are not the ones you supported initially, e.g. some ROOT classes?

Reply:
I've added `TObject`, `TNamed` and `TH1F`. Cannot think of other useful tests (?)